packages/common/src/AncillaryDataParser.ts

```diff
@@ -0,0 +1,256 @@
+import Web3 from "web3";
+import dotenv from "dotenv";
+dotenv.config();
```

Comment:
Sorry, seems like I left dotenv when writing this locally. Is it still needed as part of common utils?

Reply:
Nope, looks like we can drop it.